src/KeyDB.php

```diff
@@ -3,6 +3,26 @@
 
 namespace Tuf;
 
+/**
+ * Represent a collection of keys and their organization.  This module ensures
+ * the layout of the collection remain consistent and easily verifiable.
+ * Provided are functions to add and delete keys from the database, retrieve a
+ * single key, and assemble a collection from keys stored in TUF 'Root'
+ * Metadata. The Update Framework process maintains a set of role info for
+ * multiple repositories.
+ *
+ * RSA keys are currently supported and a collection of keys is organized as a
+ * dictionary indexed by key ID.  Key IDs are used as identifiers for keys
+ * (e.g., RSA key).  They are the hexadecimal representations of the hash of
+ * key
+ * objects (specifically, the key object containing only the public key).  See
+ * 'rsa_key.py' and the '_get_keyid()' function to learn precisely how keyids
+ * are generated.  One may get the keyid of a key object by simply accessing
+ * the
+ * dictionary's 'keyid' key (i.e., rsakey['keyid']).
```

Comment:
So, we chose to support `ed25519` keys instead of RSA. Is there anything in the code that we'll need to change to account for that, or have we already (so it's just a matter correcting the docs to reflect that)? /cc @mbaynton 

Relatedly, I think it would be a good idea for any place we're doing signing verification to take the encryption strategy as an argument, so other strategies could be supported easily in the future even though the initial implementation doesn't.

Reply:
Yeah we shouldn't talk about RSA support in the comments.

The signature verification function already takes the algorithm type, even though we currently only support one.

Reply:
Cleaned up the docs including removing these references